datasets/us/doh_debarred/us_doh_debarred.yml

```diff
@@ -0,0 +1,55 @@
+name: us_doh_debarred
+title: Debarred people and companies from us department of health
+entry_point: crawler.py
+coverage:
+  frequency: daily
+  start: "2024-03-13"
+summary: >
+  A list of debarred peoples and companies listed in US Department of Health and Human Service.
+description: |
+  This dataset provides information on Individuals and entities debarred from participation in Medicare,
+  Medicaid, and all other Federal health care programs for various reasons including conviction for Medicare 
+  or Medicaid fraud.
+  Debarments are imposed under the authority of sections 1128 and 1156 of the Social Security Act.
+  
+  From the OIG HHS page:
+  > Exclusion from participation in Medicare, Medicaid, and all other Federal health care programs.
+  > Excluded receive no payment from Federal health care programs for any items or services they
+  > furnish,order, or prescribe.
+  >
+  > Anyone who hires an individual or entity on the LEIE may be subject to civil monetary penalties (CMP).
+  > It affect ability to claims payment for health services they provide but doesn't affect 
+  > their right to receive benefits under Federal Health care programs.
+  >
+  > For exclusions implemented before August 4, 1997, the exclusion covers only the following Federal health care programs:
+  > Medicare, Medicaid, Maternal and Child Health Services Block Grant, Block Grants to States for Social Services, and
+  > State Children's Health Insurance programs.
+  >
+  > Individuals and entities who have been reinstated are removed from the LEIE.
+
+  For more information, refer to the [OIG's Exclusion Authorities FAQ page](https://oig.hhs.gov/faqs/exclusions-faq).
+
+publisher: 
+  name: U.S. Department of Health and Human Services Office of Inspector General
+  description: |
+   OIG's LEIE provides information to the health care industry, patients and the public regarding 
+   individuals and entities currently excluded from participation in Medicare, Medicaid and all other
+   Federal health care programs.
+  country: us
+  url: https://oig.hhs.gov/exclusions/index.asp
+  official: true
+url: https://oig.hhs.gov/exclusions/exclusions_list.asp
+data:
+  url: https://oig.hhs.gov/exclusions/downloadables/UPDATED.csv
+  format: CSV
+
+assertions:
+  min:
+    schema_entities:
+      Person: 10000
+      Company: 1500
+    countries: 1
+  max:
+    schema_entities:
+      Person: 150000
+      Company: 10000
```

Comment:
looking at the generated statistics.json, it looks like you can bring the min within 10% and max within 20% of the 75800 Persons and 3284 Companies